webapp/pages/profile/_id/_slug.vue

```diff
@@ -242,6 +246,7 @@ import { muteUser, unmuteUser } from '~/graphql/settings/MutedUsers'
 import { blockUser, unblockUser } from '~/graphql/settings/BlockedUsers'
 import PostMutations from '~/graphql/PostMutations'
 import UpdateQuery from '~/components/utils/UpdateQuery'
+import UserProfileHeader from '~/components/_new/generic/UserProfileHeader/UserProfileHeader'
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-06-25T11:53:18Z" title="Thursday, June 25th 2020, 1:53:18 pm +02:00">Jun 25, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -550,8 +550,8 @@ export default {
   }
 }
 
-.profile-header {
+.profile-header-card {
   padding: 0px; /* Overwrite default card padding to 0. */
-  height: 250px;
+  max-height: 250px;
```


The last changes here and/or in `webapp/pages/profile/_id/_slug.vue` made the whole header image area and dropZone invisible, unfortunately.

I haven’t investigated why. 😬